### PR TITLE
remove uniqueness from ImportBatch.name field

### PR DESCRIPTION
**Pull request by a-gleeson**
# TP2000934 remove uniqueness from ImportBatch name field

## Why
- When importing a file users should be able to use the same file name for different ImportBatches

## What
- remove uniqueness constraint on name field for ImportBatch
- use ImportBatch.pk to refer to a specific batch
- remove unique check on commodity file import field

### CODE REVIEW

File: commodities/tasks.py [hunk 1]
```diff
@@ -20,20 +17,11 @@ def run_batch_task(
     """Wraps the run_batch function in a celery task and updates the batch's
     status once complete."""
 
-    batch = models.ImportBatch.objects.get(pk=batch_pk)
-
     run_batch(
-        batch=batch,
+        batch_id=batch_pk,
```
Comment (paulpepper-trade) [inline commodities/tasks.py:21]: Does anything ever call this code (via its wrapping function)?

File: importer/forms.py [hunk 2]
```diff
@@ -201,17 +201,17 @@ def clean(self):
...
+    #     uploaded_taric_file = super().clean_taric_file()
+    #     # Because none of BatchImport's model fields are taken directly from the
+    #     # form, we can't use a ModelForm and therefore can't rely upon the
+    #     # BatchImport.name model field constraint. So do that validation here.
+    #     if ImportBatch.objects.filter(name=uploaded_taric_file.name):
+    #         raise ValidationError(
+    #             f"The uploaded file's name must be unique - "
+    #             f"{uploaded_taric_file.name} was previously uploaded.",
+    #         )
+    #     return uploaded_taric_file
```
Comment (paulpepper-trade) [inline importer/forms.py:214]: Remove this if it's no longer required?

File: importer/management/commands/chunk_taric.py [hunk 3]
```diff
@@ -36,7 +36,7 @@ def setup_batch(
 
     for dependency in dependencies or []:
         models.BatchDependencies.objects.create(
-            depends_on=models.ImportBatch.objects.get(name=dependency),
+            depends_on=models.ImportBatch.objects.get(pk=dependency),
```
Comment (paulpepper-trade) [inline importer/management/commands/chunk_taric.py:39]: Maybe worth updating the type hint, function docstring and name of `dependency` to better document it?

File: importer/tests/management/commands/test_run_import_batch.py [hunk 4]
```diff
@@ -44,7 +44,9 @@ def test_help(self, capsys):
 
         out = capsys.readouterr().out
 
-        assert "batch                 The batch Id to be imported" in out
+        print(out)
```
Comment (paulpepper-trade) [inline importer/tests/management/commands/test_run_import_batch.py:47]: Debugging leftovers?

File: importer/forms.py (hunk 2 above)
Comment (a-gleeson) [reply importer/forms.py:214]: whoops yeah this can be removed

File: commodities/tasks.py (hunk 1 above)
Comment (a-gleeson) [reply commodities/tasks.py:21]: nope nothing calls this task this now

File: importer/management/commands/chunk_taric.py (hunk 3 above)
Comment (a-gleeson) [reply importer/management/commands/chunk_taric.py:39]: already updated hint but I'll cehck for docstring updates and varaible name

File: commodities/tasks.py (hunk 1 above)
Comment (paulpepper-trade) [reply commodities/tasks.py:21]: Maybe ditch it and the file that contains it?

**Review by paulpepper-trade — APPROVED** [review]

File: commodities/tasks.py (hunk 1 above)
Comment (a-gleeson) [reply commodities/tasks.py:21]: I can remove

**Pull request merged**